src/mlpack/core/dists/gamma_distribution.cpp

```diff
@@ -0,0 +1,76 @@
+/**
+ * @file gamma_distribution.cpp
+ * @author Yannis Mentekidis
+ *
+ * Implementation of the methods of GammaDistribution.
+ */
+#include "gamma_distribution.hpp"
+
+#include <boost/math/special_functions/digamma.hpp>
+#include <boost/math/special_functions/trigamma.hpp>
```

Comment:
Maybe it is from a newer version of Boost.  You can download the different versions to pinpoint when it became available.  If we have to upgrade our required Boost version, we can probably do that, or possibly backport the digamma and trigamma distributions if needed.